template/workflows/manifests/.github/workflows/azure-kubernetes-service.yml

```diff
@@ -102,6 +101,13 @@ jobs:
           admin: 'false'
           use-kubelogin: 'true'
 
+      # Checks if the AKS cluster is private
+      - name: Is private cluster
+        id: isPrivate
+        run: |
```

Comment:
implementation looks good but we should probably have integration tests for this too, right?

Reply:
Since Draft tests don't currently support creating clusters within an Azure environment, a test for this az cluster dependent action can't be written. However, DevHub tests will run these workflows down the line and they'd get tested there

Reply:
gotcha ok, that makes sense. I imagine this has been manually tested? If so would be good to describe how you did that in the PR description